packages/filebrowser-extension/src/index.ts

```diff
@@ -394,7 +376,9 @@ const browserWidget: JupyterFrontEndPlugin<void> = {
     settings: ISettingRegistry,
     toolbarRegistry: IToolbarWidgetRegistry,
     translator: ITranslator,
-    labShell: ILabShell
+    labShell: ILabShell,
+    fileBrowserCommands: null,
```

Comment:
Let's add a comment here to clarify this trick

```suggestion
    // Wait until file browser commands are ready before activating file browser widget
    fileBrowserCommands: null,
```